app/api/helpers/import_helpers.py

```diff
@@ -294,7 +294,7 @@ def create_service_from_json(task_handle, data, srv, event_id, service_ids=None)
     return ids
 
 
-def import_event_json(task_handle, zip_path):
+def import_event_json(zip_path, task_handle, creator_id):
```

Comment:
This is unnecessary diff. There was no need to change the order here. Just add `creator_id` as last param.